apps/chatbot/src/modules/vector_database.py

```diff
@@ -30,6 +34,38 @@
     endpoint_url=f"https://s3.{AWS_DEFAULT_REGION}.amazonaws.com" if AWS_DEFAULT_REGION else None
 )
 
+LIVE_HTML = [
+    "build-devp/out/app-io/api/app-io-main.html",
+    "build-devp/out/case-histories/tari-cagliari.html",
+    "build-devp/out/firma-con-io/api/firma-con-io-main.html",
+    "build-devp/out/index.html",
+    "build-devp/out/pago-pa/api/elenco-IBAN-stazioni.html",
+    "build-devp/out/pago-pa/api/flussi-di-rendicontazione.html",
+    "build-devp/out/pago-pa/api/gestione-massiva-delle-posizioni-debitorie.html",
+    "build-devp/out/pago-pa/api/gestione-posizioni-debitorie.html",
+    "build-devp/out/pago-pa/api/gpd-fdr.html",
+    "build-devp/out/pago-pa/api/gpd-recupero-receipt.html",
+    "build-devp/out/pago-pa/api/inserimento-posizioni-debitorie.html",
+    "build-devp/out/pago-pa/api/pagamento-fe-ec.html",
+    "build-devp/out/pago-pa/api/recupero-receipt.html",
+    "build-devp/out/pago-pa/api/stampa-avvisi-pagamento.html",
+    "build-devp/out/privacy-policy.html",
+    "build-devp/out/send/api/send-main.html",
+    "build-devp/out/solutions/multe-per-violazioni-al-codice-della-strada.html",
+    "build-devp/out/solutions/tassa-sui-rifiuti-tari.html",
+    "build-devp/out/terms-of-service.html",
+    "build-devp/out/webinars.html",
+    "build-devp/out/webinars/DevTalk-pagoPA-gpd-massive.html",
+    "build-devp/out/webinars/DevTalk-pagoPA-gpd.html",
+    "build-devp/out/webinars/DevTalk-remote-content.html",
+    "build-devp/out/webinars/PagoPA-multe.html",
+    "build-devp/out/webinars/PagoPALAB-nidi.html",
+    "build-devp/out/webinars/PagoPALab-tari.html",
+    "build-devp/out/webinars/devTalks-pagoPA-fdr.html",
+    "build-devp/out/webinars/devTalks-send-essential.html",
+    "build-devp/out/webinars/nuove-api-io.html"
```

Comment:
If the documentation path can be again retrieved from params, I would interpolate it in these strings